web/client/localConfig.json

```diff
@@ -359,7 +359,12 @@
                     "showAddAsAnnotation": true
                   }
                 }
-            }, "Print", "MapImport", "MapExport", {
+            }, "Print", "MapImport", {
+                "name": "MapExport",
+                "cfg": {
+                    "enabledFormats": ["mapstore2", "wmc"]
```

Comment:
I should not enable wmc export for mapstore, but only for georchestra customization. @tdipisa what do you think about it. 
Now it looks like this: 
![image](https://user-images.githubusercontent.com/1279510/75874897-f7522700-5e12-11ea-8629-f9f701008f43.png)

Note: importing a map in wmc causes the map to be replaced, so we have an empty background and wms layers only.

Reply:
ok, we keep it